content/200-concepts/100-components/01-prisma-schema/07-referential-actions.mdx

```diff
@@ -0,0 +1,329 @@
+2. Define custom referential actions on your data model relation fields. You then update the database schema by using [Prisma Migrate](../prisma-migrate) or the [`db push`](../prisma-migrate/db-push) command.
+3. Apply the new default referential actions, that exist when no custom actions are added manually, by using [Prisma Migrate](../prisma-migrate), or [`db push`](../prisma-migrate/db-push), without making any changes to your existing Prisma schema.
+
+To begin using referential actions you add them to the `@relation` attribute in your schema.
+
+```prisma
+@relation(fields: [authorId], references: [id], onDelete: Cascade)
+```
+
+See [Upgrade guides](#upgrade-guides) for more information on adopting the feature in your project.
+
+</TopBlock>
+
+## What are referential actions?
+
+Referential actions are constraints that can be used to define how a referenced record is handled by the database when either an [`update`](../prisma-client/crud#update) or [`delete`](../prisma-client/crud#delete) query is run.
+
+These include:
+
+- [`Cascade`](#cascade)
+- [`Restrict`](#restrict)
+- [`NoAction`](#noaction)
+- [`SetNull`](#setnull)
+- [`SetDefault`](#setdefault)
+
+To understand how referential actions can impact your data it is important to understand how foreign key constraints work.
+
+[Foreign keys](../../../guides/general-guides/database-workflows/foreign-keys#overview) are used to link fields between tables and represent [relations](./relations) in your database.
+
+The following example shows `User` and `Post` models with a [one-to-many](./relations#one-to-many-relations) relationship. The `Post` model has an `authorId` field which is mapped to a foreign key constraint on the database table.
+
+```prisma file=schema.prisma
+model Post {
+  id        Int     @id @default(autoincrement())
+  title     String
+  author    User    @relation(fields: [authorId], references: [id])
+  authorId  Int
+}
+
+model User {
+  id    Int         @id @default(autoincrement())
+  posts Post[]
+}
+```
+
+The foreign key ensures that when saving rows into the `Post` table, any value in the `authorId` [scalar field](./data-model#scalar-fields) must be referencing an existing value in the `id` column of the `User` table. 
+
+When a [migration](../prisma-migrate) is run the following SQL is produced:
+
+```sql
+CREATE TABLE "Post" (
+    "id" INTEGER NOT NULL PRIMARY KEY AUTOINCREMENT,
+    "title" TEXT NOT NULL,
+    "authorId" INTEGER NOT NULL,
+    FOREIGN KEY ("authorId") REFERENCES "User" ("id") ON DELETE RESTRICT ON UPDATE CASCADE
+);
+
+CREATE TABLE "User" (
+    "id" INTEGER NOT NULL PRIMARY KEY AUTOINCREMENT
+);
+```
+
+And the foreign key relation that was created can be seen here:
+
+```sql
+FOREIGN KEY ("authorId") REFERENCES "User" ("id") ON DELETE RESTRICT ON UPDATE CASCADE
+```
+
+### Behavior without the preview feature flag `referentialActions`
```

Comment:
I think we should move this (and all references to pre-2.26.0 generally) to the bottom of the page, including remove the pre-2.26.0 columns from the tables. If someone is coming to this page who isn't using this preview, they have everything they need in one section, deprioritized because they can only choose to use the defaults or use raw SQL - perhaps with a link at the very top that says: "See also: Referential actions before 2.25.0"

Reply:
I still think the pre 2.26 column serves a purpose by giving a quick overview of how it was before.